fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentImplicits.scala

```diff
@@ -0,0 +1,119 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.{Element, Tag, Content}
+import com.gu.facia.api.models.{ImageReplace, ImageCutout, FaciaContent}
+import org.joda.time.DateTime
+
+import scala.util.Try
+
+object FaciaContentImplicits {
+
+  implicit class FaciaContentImplicit(fc: FaciaContent) {
```

Comment:
Do we really need both ways of accessing these? Is it not better to encourage users of this library to pick one way or the other? It means there's the burden of keeping these in sync forever more and precisely one of the approaches is redundant.